doc/source/releases/21.09_announce.rst

```diff
@@ -125,7 +125,6 @@ config/galaxy.yml.sample:galaxy
 -  job_runner_monitor_sleep
 -  mailing_join_body
 -  mailing_join_subject
--  mailing_lists_url
```

Comment:
Please revert, no reason to change old release notes unless they are factually wrong.